cypress/integration/links_test.js

```diff
@@ -0,0 +1,94 @@
+describe("My First Test", () => {
+  beforeEach(() => {
+    cy.visit("https://docs.lob.com/");
+  });
+
+  it("contains a link to the main Lob page", () => {
+    // makes sure the lob.com link goes to the right page
+    cy.get('*[class^="brand w-nav-brand"]')
+      .should("have.attr", "href")
+      .and("include", "https://lob.com");
+  });
+
+  it("contains the support link", () => {
+    // makes sure the support link goes to the right page
+    cy.get('*[class^="sc-gsWcmt gCgqqY"]')
```

Comment:
Depending on your build system and what it's configured to do, these classes may be randomly generated each time - if that's the case, this may fail after the first time running the tests. 😅  if it's the same random classes each time (if that statement makes sense, lol), this should be fine though!

Reply:
Thank you for bringing this up! Actually, the classes are generated by Redocly and they're static, which is why I figured it's safe enough to do this. I've verified that they don't change from build to build, so we should be good (even though it's a little frowned-upon in Cypress).

Reply:
Got it. I would still recommend using HTML tags/other means whenever you can (again, because if someone somewhere decides they want a different class on a thing, your tests will break despite actual functionality not changing, whereas HTML tags should almost always remain the same unless you're actually changing what's on the screen), but again, using classes is ... fine. 🙃

Reply:
Based on our conversation, there are two main reasons which make me think that classes are the best option after all:

(1) The Redocly-generated class names are static and do not regenerate between deploys. Even changing the colour of an element is done by referencing the Redocly class name in our `template` Handlebars file.

(2) Not all of the links being tested are attached to elements which contain text, so the `get().contains()` chain doesn't always work. For example, there's one link which is associated with an image, so I can't use `contains()` to find that––but it does have a unique, static class name.